### ISSUE TITLE
Accessibility Icon

### ISSUE THREAD

**Issue by colabottles** [simple-icons/simple-icons#12482]
### Brand Name

Accessibility

### Website

https://w3.org

### Popularity Metric

The accessibility icon should be added to support the accessibility community.

The Similarweb rank is 3,641,221 

The icon is used worldwide.

### Official Resources for Icon and Color

https://accessibleicon.org/

**Comment by LitoMore** [comment]
Duplicate of #7103.
